api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/TxCurrCohortQueries.java

```diff
@@ -246,30 +249,12 @@ public CohortDefinition getPatientsWithNextConsultationDate() {
 	}
 	
 	/**
-	 * Build TxCurr composition cohort definition
-	 * 
-	 * @param cohortName
-	 * @param inARTProgramAtEndDate
-	 * @param patientWithSTARTDRUGSObs
-	 * @param patientWithHistoricalDrugStartDateObs
-	 * @param patientsWithDrugPickUpEncounters
-	 * @param patientsWhoLeftARTProgramBeforeOrOnEndDate
-	 * @param patientsThatMissedNexPickup
-	 * @param patientsReportedAsAbandonmentButStillInPeriod
-	 * @param ageCohort
-	 * @param patientsWithNextPickupDate
-	 * @param patientsWithNextConsultationDate
+	 * @param cohortName Cohort name
 	 * @param currentSpec
-	 * @return CompositionQuery
+	 * @return TxCurr composition cohort definition
 	 */
 	@DocumentedDefinition(value = "getTxCurrCompositionCohort")
-	public CohortDefinition getTxCurrCompositionCohort(String cohortName, CohortDefinition inARTProgramAtEndDate,
-	        CohortDefinition patientWithSTARTDRUGSObs, CohortDefinition patientWithHistoricalDrugStartDateObs,
-	        CohortDefinition patientsWithDrugPickUpEncounters, CohortDefinition patientsWhoLeftARTProgramBeforeOrOnEndDate,
-	        CohortDefinition patientsThatMissedNexPickup, CohortDefinition patientsThatMissNextConsultation,
-	        CohortDefinition patientsReportedAsAbandonmentButStillInPeriod, CohortDefinition ageCohort,
-	        CohortDefinition genderCohort, CohortDefinition patientsWithNextPickupDate,
-	        CohortDefinition patientsWithNextConsultationDate, boolean currentSpec) {
+	public CohortDefinition getTxCurrCompositionCohort(String cohortName, boolean currentSpec) {
 		
 		final int abandonmentDays = currentSpec ? CURRENT_SPEC_ABANDONMENT_DAYS : OLD_SPEC_ABANDONMENT_DAYS;
 		CompositionCohortDefinition TxCurrComposition = new CompositionCohortDefinition();
```

Comment:
Generally, all variables name should start with lower case, @here rename TxCurrComposition to txCurrComposition